test/LdapTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
```

Comment:
I think the test suite needs an analisys and may a complete refactor and avoid this kind of confusions.

Could you last the following tasks before to merge?

1. Move the contents of this file to OfflineTest.php
2. Squash the PR (rewrite the history to only one commit)